management/server/peer.go

```diff
@@ -312,17 +314,22 @@ func (am *DefaultAccountManager) GetPeerByIP(accountID string, peerIP string) (*
 }
 
 // GetNetworkMap returns Network map for a given peer (omits original peer from the Peers result)
-func (am *DefaultAccountManager) GetNetworkMap(peerPubKey string) (*NetworkMap, error) {
+func (am *DefaultAccountManager) GetNetworkMap(peerID string) (*NetworkMap, error) {
 
-	account, err := am.Store.GetAccountByPeerPubKey(peerPubKey)
+	account, err := am.Store.GetAccountByPeerID(peerID)
 	if err != nil {
 		return nil, err
 	}
 
-	aclPeers := account.getPeersByACL(peerPubKey)
-	routesUpdate := account.getRoutesToSync(peerPubKey, aclPeers)
+	peer := account.GetPeer(peerID)
```

Comment:
it might be better to return an error from account.GetPeer, generally speaking, it should never happen, but if it does, let's centralize it in one method?

Reply:
After discussing internally, we are not returning error from the method because of different cases where a not found peer might cause a different kind of error to the APi, e.g. not found is ok for peer deletion, but for route creation no.